Fix error importing package

If I try to import the package using:

```typescript
const { wren } = await import('@exercism/codemirror-lang-wren')
```

I get the following error:

```
TS2307: Cannot find module '@exercism/codemirror-lang-wren' or its corresponding type declarations.
```

Changing the dynamic import to a regular one results in the same error:

```typescript
import { wren } from '@exercism/codemirror-lang-wren'
```

Importing a file using `import '@exercism/codemirror-lang-wren/dist/index.es'` _does_ work.

The `node_modules/@exercism/codemirror-lang-wren` directory has the following files:

```
dist/index.cjs
dist/index.es.js
LICENSE
package.json
README.md
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,0 +1,7 @@
+import { Completion } from "@codemirror/autocomplete"
+import { LanguageSupport, LezerLanguage } from "@codemirror/language"
+
+export const wrenLanguage: LezerLanguage
+export function wren(config?: {}): LanguageSupport
+export const snippets: Completion[]
+

--- a/package.json
+++ b/package.json
@@ -14,6 +14,7 @@
     "import": "./dist/index.es.js",
     "require": "./dist/index.cjs"
   },
+  "types": "index.d.ts",
   "scripts": {
     "test": "mocha test/test-*.js",
     "prepare": "npm run build",
@@ -36,6 +37,7 @@
     "lezer": "^0.13.5"
   },
   "files": [
-    "dist"
+    "dist",
+    "index.d.ts"
   ]
 }

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "@exercism/codemirror-lang-wren",
   "description": "Wren language support for the CodeMirror code editor",
   "author": "Josh Goebel <me@joshgoebel.com>",
-  "version": "0.5.0",
+  "version": "0.6.0",
   "license": "MIT",
   "repository": {
     "type": "git",

--- a/package.json
+++ b/package.json
@@ -10,10 +10,6 @@
   },
   "main": "dist/index.cjs",
   "module": "dist/index.es.js",
-  "exports": {
-    "import": "./dist/index.es.js",
-    "require": "./dist/index.cjs"
-  },
   "types": "index.d.ts",
   "scripts": {
     "test": "mocha test/test-*.js",

--- a/package.json
+++ b/package.json
@@ -10,6 +10,11 @@
   },
   "main": "dist/index.cjs",
   "module": "dist/index.es.js",
+  "exports": {
+    "import": "./dist/index.es.js",
+    "require": "./dist/index.cjs"
+  },
+  "type": "module",
   "types": "index.d.ts",
   "scripts": {
     "test": "mocha test/test-*.js",

--- a/package.json
+++ b/package.json
@@ -28,12 +28,12 @@
     "@codemirror/language": "^0.18.1"
   },
   "devDependencies": {
-    "@codemirror/basic-setup": "^0.18.1",
+    "@codemirror/basic-setup": "^0.18.2",
     "@rollup/plugin-node-resolve": "^9.0.0",
-    "lezer-generator": "^0.13.0",
-    "mocha": "^8.1.3",
-    "rollup": "^2.27.1",
-    "lezer": "^0.13.0"
+    "lezer-generator": "^0.13.4",
+    "mocha": "^8.4.0",
+    "rollup": "^2.50.4",
+    "lezer": "^0.13.5"
   },
   "files": [
     "dist"

